spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -1563,6 +1675,40 @@ public void setRotate(boolean rotate) {
 
 		}
 
+		/**
+		 * Undertow thread properties.
+		 */
+		public static class Threads {
+
+			/**
+			 * Number of I/O threads to create for the worker. The default is derived from
+			 * the number of available processors.
+			 */
+			private Integer io;
```

Comment:
`io` is a bit odd on its own. I don't have bright ideas but perhaps we should add something in the property name that better conveys it's a number of threads for a certain "type" of threads.

Reply:
Yep I struggled w/ this balance of brevity and better conveyed meaning. For this one I was thinking `io-per-worker` as that is actually what it is.